docs/ACCEL.md

```diff
@@ -64,12 +64,14 @@ Currently Zoom relies partially on Metal and so needs a small binary patch. Dosd
 * [Zoom Non-Metal Fix](http://dosdude1.com/catalina/zoomnonmetal-new.command.zip)
 
 ## Unable to grant special permissions to apps (ie. Camera Access to Zoom)
+Currently for Ventura 13.3 and newer, due to new patches required, permissions are yet again broken. 
```

Comment:
Keep the additional information that was formerly in the `0.2.4 and older Work-Around` sections

Reply:
Done 👍